### DISCUSSION TITLE
Show and tell... your stats!

### DISCUSSION THREAD

**Discussion by H4waiianPunch** [H4waiianPunch/Pearl-Luck-Tracker#16] [Show and tell]
Moved this over from #10, makes more sense.

Hey everyone,

I thought it would be fun to show off all your stats at the end of your Aerial Fishing grind! It'll be interesting to see how many Golden Tenches some people get etc!

**Comment by ben-sharp** [comment]
Early on the grind only 100 pearls but biggest spoon so far is 6!

**Comment by nunny0206** [comment]
Rates going well so far! Thanks for the plugin, the tench chance is technically probability.
Also the tench rate resets per session, but I would think it should reset each time you get a tench, instead of for each session
<img width="270" height="409" alt="image" src="https://github.com/user-attachments/assets/06d84a9f-9bbd-4180-880b-4d91e6050e5b" />

**Reply by nunny0206** [reply]
<img width="280" height="424" alt="image" src="https://github.com/user-attachments/assets/0f5224ed-1f6a-4bbf-94af-e90503d9ff3c" />

**Reply by nunny0206** [reply]
3rd tench at 40k! Like having an extra one for free so far
<img width="276" height="416" alt="image" src="https://github.com/user-attachments/assets/dff6c4d4-d6cc-4058-a333-330b4ad7ef25" />

**Reply by nunny0206** [reply]
4th Tench and the chance reset to 0.0%, was that a hotfix or was i just wrong before? :)
<img width="216" height="339" alt="image" src="https://github.com/user-attachments/assets/ed56fe2f-8940-49ee-b525-60e50b80ea51" />

**Reply by nunny0206** [reply]
wait it came back!
<img width="219" height="333" alt="image" src="https://github.com/user-attachments/assets/1c9ffa40-c327-4c5c-afd5-80aec92a03fa" />

**Reply by H4waiianPunch** [reply]
HUH. Interesting.. I thought I had made it reset upon getting a tench but I guess not..? I also am not sure why it went to 0% and then back to 83%... Something to maybe look into I suppose!

**Reply by nunny0206** [reply]
Made it to my fish sack, will come back one day for rods I'm sure :)
Thanks again for the mod
<img width="408" height="617" alt="image" src="https://github.com/user-attachments/assets/c3ffd8de-acad-444e-aeb3-7ea1cd4e792d" />
